client/my-sites/domains/controller.jsx

```diff
@@ -30,8 +30,7 @@ import { getCurrentUser } from 'state/current-user/selectors';
  */
 const productsList = productsFactory();
 
-module.exports = {
-
+export default {
```

Comment:
I don't care about it in terms of size, but how it affects structure of the file. Those are "root" functions, they should not be indented. Plus, it makes it harder to read on smaller screens, forces us to use shorter lines, etc.